src/yaxarrays/YAXArrays.jl

```diff
@@ -0,0 +1,36 @@
+import Base.copy
+
+function axes_names(cube::YAXArray)
+    return name.(cube.axes)
+end
+
+"""
+    sort(cube::YAXArray, axis_name::Symbol)
+
+Sorts axis labels of a given YAXArray datacube.
+"""
+function sort(cube::YAXArray, axis_name::Symbol)::YAXArray
+    axis_labels = collect(lookup(cube, axis_name))
+    labels_sort_idx::Vector{Int64} = sortperm(axis_labels)
+    ordered_labels = axis_labels[labels_sort_idx]
+
+    # Get selector to order data
+    _axes_names = axes_names(cube)
+    n_axes = length(_axes_names)
+    selector::Vector{Union{Colon,Vector{Int64}}} = fill(:, n_axes)
+    axis_idx::Int64 = findfirst(x -> x == axis_name, _axes_names)
+    selector[axis_idx] = labels_sort_idx
+
+    return cube[selector...]
+end
+
+"""
+    copy(cube::YAXArray)::YAXArray
+
+Use julia's Base.copy function to create a copy of an YAXArray
```

Comment:
This statement is technically incorrect - you're not using Base.copy.

Better to state what the method does / is for.

Reply:
Hum, inside this function I'm using Base.copy to create a copy of the data inside the YAXArray. I can rewrite this to better explain what its being done there.

Reply:
Oh I see - still, probably just something succinct like "copy a data cube" is fine